pom.xml

```
@@ -44,7 +44,7 @@
     <properties>
         <liquibase.version>4.33.0</liquibase.version>
         <spring-boot.version>3.5.5</spring-boot.version>
-        <spring-data-opensearch.version>1.8.1</spring-data-opensearch.version>
+        <spring-data-opensearch.version>2.0.0</spring-data-opensearch.version>
```

If you don't mind, we just dropped `2.0.1`:

```suggestion
        <spring-data-opensearch.version>2.0.1</spring-data-opensearch.version>
```